main.js

```diff
@@ -246,11 +246,5 @@ async function processCommand(receivedMessage){
         case "credits":
             bootstrap.OracleObj.credits(receivedMessage, arguments);
             break;
-        default:
```

Comment:
While we don't want the user to see whether or not the bot couldn't recognize the command, it would probably be good to log it.  
For the default: condition, add a log that says what the command was that couldn't find a match.  This can help with debugging later on.